com.unity.cinemachine/Samples~/Input System/Split Screen Multiplayer/CinemachineCameraSetter.cs

```diff
@@ -0,0 +1,19 @@
+using UnityEngine;
+
+namespace Unity.Cinemachine.Samples
+{
+    public class CinemachineCameraSetter: MonoBehaviour
+    {
+        [SerializeField] CinemachineBrain m_CinemachineBrain;
+        [SerializeField] CinemachineCamera m_CinemachineCamera;
+        void Start()
+        {
+            // Increment to the next channel based on the brain count for the CinemachineBrain and the CinemachineCamera.
```

Comment:
This comment doesn't match the code.  Here you're initializing the start position, not selecting a channel

Reply:
Changed